src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedForDepthCheckTest.java

```diff
@@ -44,7 +44,7 @@ protected String getPackageLocation() {
      *     interface-requirements for test-methods.
      */
     @Test
-    public void testNestedTooDeep() throws Exception {
+    public void testNestedForDepthLevelTwo() throws Exception {
```

Comment:
Please do rename the test method name according to new inputs file name
testNestedForDepthLevelTwo --> `testNestedForDepthCheckCustomMaxLevelTwo`